lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.ts

```diff
@@ -0,0 +1,276 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, Input, OnInit, EventEmitter, Output } from '@angular/core';
+import {
+    CardViewDateItemModel,
+    CardViewItem,
+    CardViewTextItemModel,
+    CardViewBaseItemModel,
+    TranslationService,
+    AppConfigService,
+    CardViewMapItemModel,
+    UpdateNotification,
+    CardViewUpdateService,
+    StorageService
+} from '@alfresco/adf-core';
+import { TaskHeaderCloudService } from '../services/task-header-cloud.service';
+import { TaskDetailsCloudModel } from '../../start-task/models/task-details-cloud.model';
+
+@Component({
+    selector: 'adf-cloud-task-header',
+    templateUrl: './task-header-cloud.component.html',
+    styleUrls: ['./task-header-cloud.component.scss']
+})
+export class TaskHeaderCloudComponent implements OnInit {
+
+    /** (Required) The appName */
+    @Input()
+    appName: string;
+
+    /** (Required) The id of the task. */
+    @Input()
+    taskId: string;
+
+    /** The name of the form. */
+    @Input()
+    formName: string = null;
```

Comment:
for my point of view, this should not be an input property but it should be fetched with the taskDetails